doc/news/_preparation_next_release.md

```diff
@@ -149,6 +149,7 @@ The following text lists news about the [plugins](https://www.libelektra.org/plu
 - Remove duplicated link to `TESTING.md` file _(@stefnotch)_, _(@janldeboer)_
 - <<TODO>>
 - Restructure parts of the links on the website _(@stefnotch)_, _(@janldeboer)_
+- Fixed broken Links with a link to the Mainpage _(@Dynamichost96)_
```

Comment:
```suggestion
- Removed broken links to packages for Linux distributions. _(@Dynamichost96)_
```

Reply:
The text in the release notes has to be updated to correctly describe the actual changes in the PR.

Reply:
Thanks, i changed the release note.